src/SlashCommands.tsx

```diff
@@ -437,7 +446,11 @@ export const Commands = [
                 return success(cli.setRoomTopic(roomId, args));
             }
             const room = cli.getRoom(roomId);
-            if (!room) return reject(_t("Failed to set topic"));
+            if (!room) {
+                return reject(
+                    newTranslatableError("Failed to set topic: Unable to find room (%(roomId)s", { roomId }),
```

Comment:
Added more info on why something failed for the few places we check for `if (!room)`

Reply:
This is not right, this is to *view* the topic. The *set* path is the `if (args) { ... }` one above